build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -319,10 +341,12 @@ String _dartLoaderSetup(Map<String, String> modulePaths, String appDigests,
     '''
 $_currentDirectoryScript
 $_baseUrlScript
+let rootDirectory = window.location.origin + baseUrl;
 let modulePaths = ${const JsonEncoder.withIndent(" ").convert(modulePaths)};
+let appDigests = _currentDirectory + '$appDigests';
 if(!window.\$dartLoader) {
    window.\$dartLoader = {
-     appDigests: _currentDirectory + '$appDigests',
+     appDigests: new Array(),
```

Comment:
This does change the type of `window.$dartLoader.appDigests` from a string to an array of strings, but I couldn't find anything in here or in webdev that is actually using this value, so I thought it was safe.

Reply:
@annagrin do you know if anything is using this?

Reply:
I think maybe this used to be used by hot reload but we dropped the support for that in webdev? Maybe we should stop adding this altogether.

Reply:
I found this use in legacy strategy (for google3 users): https://github.com/dart-lang/webdev/blob/0274a13ffd50a77a4808c294d843ec1f7dcd06cd/dwds/lib/src/loaders/legacy.dart#L97

and that's where it is called: https://github.com/dart-lang/webdev/blob/a0b8e2b062fa8cfafc3c410259e934322cc4dde6/dwds/lib/src/handlers/injector.dart#L201

Reply:
Sorry, maybe I'm missing something - those are usages of `$dartLoader`, but not of this `appDigests` field, right? I think `$dartLoader.forceLoadModule()` would still behave the same